pkg/apis/batch/types.go

```diff
@@ -209,7 +209,7 @@ type ScheduledJobSpec struct {
 
 	// Suspend flag tells the controller to suspend subsequent executions, it does
 	// not apply to already started executions.  Defaults to false.
-	Suspend *bool `json:"suspend"`
+	Suspend *bool `json:"suspend,omitempty"`
```

Comment:
Just curious why use `*bool` here since it's an optional field and there's no different between this value be unset or false